autogen/main/variables.tf.tmpl

```diff
@@ -619,6 +619,14 @@ variable "disable_default_snat" {
   default     = false
 }
 
+{% if autopilot_cluster != true %}
+variable "enable_default_node_pools_metadata" {
+  type        = bool
+  description = "Whether to enable the default node pools metadata key-value pairs such as `cluster_name` and `node_pool`"
+  default     = true
+}
```

Comment:
nit: let's move the line return into the block so we don't get extra spaces in autopilot_cluster != true

```suggestion
}

```